entity-framework/core/miscellaneous/dbcontextpool.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Connection pooling
+description: Connection pooling in Entity Framework Core
+author: rick-anderson
+ms.author: riande
+ms.date: 9/19/2020
+uid: core/miscellaneous/dbcontextpool
+---
+# Connection pooling
+
+`AddDbContextPool` enables `DbContext` pooling. Context pooling can increase throughput in high-scale scenarios such as web servers by reusing `DbContext` instances, rather than creating new instances for each request.
+
+The typical pattern in an ASP.NET Core app using EF Core involves registering a custom <xref:Microsoft.EntityFrameworkCore.DbContext> type into the [dependency injection](/aspnet/core/fundamentals/dependency-injection) container and obtaining instances of that type through constructor parameters in controllers or Razor Pages. Using constructor injection, a new instance of the `DbContext` is created for each request.
+
+<xref:Microsoft.Extensions.DependencyInjection.EntityFrameworkServiceCollectionExtensions.AddDbContextPool%2A> enables a pool of reusable `DbContext` instances. To use `DbContext` pooling, use the `AddDbContextPool` instead of `AddDbContext` during service registration:
+
+``` csharp
+services.AddDbContextPool<BloggingContext>(
```

Comment:
As a general rule, we try to always put code samples under the samples in this repo, and reference it - this makes sure code always works and is up to date (we build the sample repo as part of the docs CI).

Reply:
@roji  do you want me to add a project to samples that just has a main that calls startup with [this code](https://github.com/dotnet/EntityFramework.Docs/blob/master/samples/core/DbContextPooling/Program.cs#L38-L49)